clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -788,6 +792,97 @@ data ResetStrategy (r :: Bool) where
   ClearOnReset :: ResetStrategy 'True
   NoClearOnReset :: ResetStrategy 'False
 
+-- | Version of blockram that has no default values set. May be cleared to a
+-- arbitrary state using a reset function.
+blockRam0
```

Comment:
I think I prefer name `blockRamU` to refer to the _U_ndefined initial state.

Reply:
Very well, done.